pkg/reconciler/autoscaling/kpa/scaler.go

```diff
@@ -87,6 +89,11 @@ type scaler struct {
 	// For async probes.
 	probeManager asyncProber
 	enqueueCB    func(interface{}, time.Duration)
+
+	// For service initial scale.
+	initialScaleTime  time.Duration
```

Comment:
not sure this matters that much, but initialScaleTime is actually a time.Duration, not a time.Time, which is a bit confusing. Might be worth renaming